react/components/Select/README.md

```diff
@@ -0,0 +1,214 @@
+    <Select
+      defaultValue={options[0]}
+      isMulti={true}
+      label="Regular"
+      options={options}
+      onChange={values => {
+        console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+      }}
+    />
+  </div>
+  <div className="mb5">
+    <Select
+      defaultValue={options[0]}
+      size="large"
+      isMulti={true}
+      label="Large"
+      options={options}
+      onChange={values => {
+        console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+      }}
+    />
+  </div>
+</div>
+```
+
+Single
+
+```js
+const options = [
+  {
+    value: 'first-option',
+    label: 'First Option',
+  },
+  {
+    value: 'second-option',
+    label: 'Second Option',
+  },
+]
+
+;<div>
+  <Select
+    label="Single option select"
+    options={options}
+    isMulti={false}
+    onChange={values => {
+      console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+    }}
+  />
+</div>
+```
+
+Multi Select
+
+```js
+const options = [{
+  value: 'first-option',
+  label: 'First option',
+},
+{
+  value: 'second-option',
+  label: 'Second option',
+}];
+
+<div>
+  <Select
+    label="Label"
+    options={options}
+    isMulti={true}
+    onChange={values => {
+      console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+    }}
+  />
+</div>
+```
+
+With Error
+
+```js
+const options = [
+  {
+    value: 'first-option',
+    label: 'First Option',
+  },
+  {
+    value: 'second-option',
+    label: 'Second Option',
+  },
+]
+
+;<div>
+  <Select
+    label="Label"
+    options={options}
+    isMulti={true}
+    errorMessage="Required!"
+    onChange={values => {
+      console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+    }}
+  />
+</div>
+```
+
+Disabled
+
+```js
+const options = [
+  {
+    value: 'first-option',
+    label: 'First Option',
+  },
+  {
+    value: 'second-option',
+    label: 'Second Option',
+  },
+]
+
+;<div>
+  <Select
+    isDisabled={true}
+    label="Label"
+    options={options}
+    isMulti={true}
+    onChange={values => {
+      console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+    }}
+  />
+  <div className="mv5">
+    <Select
+      isDisabled={true}
+      label="Label"
+      options={options}
+      isMulti={true}
+      onChange={values => {
+        console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+      }}
+      value={[
+        {
+          value: 'first-option',
+          label: 'First Option',
+        },
+      ]}
+    />
+  </div>
+</div>
+```
+
+Loading state
+
+```js
+const options = [
+  {
+    value: 'first-option',
+    label: 'First Option',
+  },
+  {
+    value: 'second-option',
+    label: 'Second Option',
+  },
+]
+
+;<div>
+  <Select
+    isLoading={true}
+    label="Label"
+    options={options}
+    isMulti={true}
+    onChange={values => {
+      console.log(`[Select] Selected: ${JSON.stringify(values, null, 2)}`)
+    }}
+  />
+</div>
+```
```

Comment:
Senti falta de um exemplo interativo, ou seja, com state e com valores selecionados sendo passados.

Vc pode fazer assim no Styleguidist:

```js
initialState = {
  value: ''
};
<Input
	value={state.value}
	onChange={(e) => {
		setState({ value: e.target.value })
	}
/>
```

Nota que nao tem o `this.`